zmq/test/zmq_test.ml

```diff
@@ -325,7 +325,7 @@ let suite =
              bind rep endpoint;
              connect req endpoint;
              subscribe sub "";
-             let mask = mask_of [| req, In_out; rep, In_out; sub, In_out |] in
+             let mask = mask_of [| (req :> [`Req | `Rep | `Sub] Socket.t), In_out; (rep :> [`Req | `Rep | `Sub] Socket.t), In_out; (sub :> [`Req | `Rep | `Sub] Socket.t), In_out |] in
```

Comment:
Would it be possible to create helper function so this line could be written as:
```ocaml
mask_of [| Poll.in_out req; Poll.in_out rep; Poll.in_out sub |]
```
So adding three functions: `in_out`, `in`[1] `out` which erases the type of the sockets?

[1]: I know that a function cannot be called `in`, so some other naming scheme is needed.

Reply:
Yep have done so, see 18b3c5c. Not sure about the name of the function, lmk if you have any better naming scheme.